docs/deploying_csi_vsphere_with_rbac.md

```diff
@@ -2,6 +2,18 @@
 
 This document is designed to quickly get you up and running with the `csi-vsphere`.
 
+## Requirements
+
+The vSphere Container Storage Interface (CSI) driver depends on First Class Disks (FCDs) and as such, has the following requirements:
```

Comment:
I would explicitly state that versions prior to vSphere 6.5 are not supported

Reply:
Will do.